sklearn/linear_model/tests/test_ridge.py

```diff
@@ -49,8 +49,11 @@
 X_iris = sp.csr_matrix(iris.data)
 y_iris = iris.target
 
-DENSE_FILTER = lambda X: X
-SPARSE_FILTER = lambda X: sp.csr_matrix(X)
+
+def DENSE_FILTER(X): return X
```

Comment:
While I said, sure, get rid of the extra `rng =`, in general, we avoid changing unrelated things in PRs. They make it harder to review and may cause merge conflicts with other work.

Reply:
Sorry, pep8 made that. Generally, I reverse those changes but missed this.